src/components/search-bar.component.ios.js

```diff
@@ -12,6 +12,13 @@ export class SearchBar extends Component {
     onSearchButtonPress: PropTypes.func.isRequired,
   };
 
+  static defaultProps = {
```

Comment:
Just realized you're using the `proptypes` package to define props in these components. The rest of the application uses flow so I think it makes sense to do the same here. You can do this by adding a simple `Props` object like so:

```
type Props = {
  textColor: string,
  textFieldBackgroundColor: string,
  showsCancelButton: bool,
  placeholder: string,
  onFocus: Function,
  onCancelButtonPress: Function,
  onSearchButtonPress: Function,
};
```

and then adding it as a type:

```
export const Searchbar = ({
  textColor,
  textFieldBackgroundColor,
  showsCancelButton,
  placeholder,
  onFocus,
  onCancelButtonPress,
  onSearchButtonPress,
}: Props) => {
```

This should save us the need to define defaultProps as well.

Also each of the components in the `components` folder are stateless functional. Although everyone has their own preferences, I think it makes sense to follow the same format here as well. Will prefer if you can make `search-bar.ios` and `search-bar.android` the same if that's okay :)